packages/belos/tpetra/example/LSQR/LSQRTpetraExFile.cpp

```diff
@@ -0,0 +1,349 @@
+  using STM = Teuchos::ScalarTraits<MT>;
+  using STS = Teuchos::ScalarTraits<ST>;
+
+  using LinearProblem = typename Belos::LinearProblem<ST, MV, OP>;
+  using Solver = ::Belos::LSQRSolMgr<ST, MV, OP>;
+  
+  Teuchos::GlobalMPISession session(&argc, &argv, NULL);
+  RCP<const Teuchos::Comm<int>> comm = Tpetra::getDefaultComm();
+
+  bool verbose = false;
+  bool success = true;
+
+  try
+  {
+    bool procVerbose = false;
+    bool debug = false;
+    int frequency = -1; // frequency of status test output.
+    int blockSize = 1;  // blockSize
+    int numRHS = 1;     // number of right-hand sides to solve for
+    int maxiters = -1;  // maximum number of iterations allowed per linear system
+    std::string filename("orsirr1_scaled.hb");
+    std::string filenameRHS; // blank mean unset
+    MT relResTol = STM::squareroot (STS::eps ());   // relative residual tolerance
+    // Like CG, LSQR is a short recurrence method that
+    // does not have the "n" step convergence property in finite precision arithmetic.
+    MT resGrowthFactor = 4.0; // In this example, warn if |resid| > resGrowthFactor * relResTol
+    // With no preconditioner, this is only the difference between the "implicit" and the "explict
+    // residual.
+
+    MT relMatTol = 1.e-4; // relative Matrix error, default value sqrt(eps)
+    MT maxCond = 1.e+8;   // maximum condition number default value 1/eps
+    MT damp = 0.;         // regularization (or damping) parameter
+
+    Teuchos::CommandLineProcessor cmdp(false, true); // e.g. ./a.out --tol=.1 --filename=foo.hb
+
+    cmdp.setOption("verbose", "quiet", &verbose, "Print messages and results.");
+    cmdp.setOption("debug", "nondebug", &debug, "Print debugging information from solver.");
+    cmdp.setOption("frequency", &frequency, "Solvers frequency for printing residuals (#iters).");
+    cmdp.setOption("filename", &filename, "Filename for test matrix.  Acceptable file extensions: *.hb,*.mtx,*.triU,*.triS");
+    cmdp.setOption("rhsFilename", &filenameRHS, "Filename for right-hand side.  Acceptable file extension: *.mtx");
+    cmdp.setOption("lambda", &damp, "Regularization parameter");
+    cmdp.setOption("tol", &relResTol, "Relative residual tolerance");
+    cmdp.setOption("matrixTol", &relMatTol, "Relative error in Matrix");
+    cmdp.setOption("max-cond", &maxCond, "Maximum condition number");
+    cmdp.setOption("num-rhs", &numRHS, "Number of right-hand sides to be solved for.");
+    cmdp.setOption("block-size", &blockSize, "Block size used by LSQR."); // must be one at this point
+    cmdp.setOption("max-iters", &maxiters, "Maximum number of iterations per linear system (-1 = adapted to problem/block size).");
+
+    if (cmdp.parse(argc, argv) != Teuchos::CommandLineProcessor::PARSE_SUCCESSFUL)
+    {
+      return -1;
+    }
+    if (!verbose)
+      frequency = -1; // reset frequency if test is not verbose
+
+    //
+    // *************Get the problem*********************
+    //
```

Comment:
Consistency in comments. Sometimes it's:
```
//
// ...
//
```

and sometimes it's: `//`
If personality thing that `//` is good

Reply:
Solved